upup/pkg/fi/cloudup/awstasks/loadbalancer_attributes.go

```diff
@@ -0,0 +1,211 @@
+/*
```

Comment:
Another documentation point. We will need Terraform support for this. Cross documenting here https://github.com/kubernetes/kops/issues/838

Lets not worry about it in this PR - we should open a new PR that adds in Terraform support for the other tasks as well

No change needed.